tfhe/Cargo.toml

```diff
@@ -61,15 +62,19 @@ console_error_panic_hook = { version = "0.1.7", optional = true }
 serde-wasm-bindgen = { version = "0.4", optional = true }
 getrandom = { version = "0.2.8", optional = true }
 bytemuck = "1.13.1"
+combine = { version = "*", optional = true }
+anyhow = { version = "*", optional = true }
 
 [features]
 boolean = []
 shortint = []
 integer = ["shortint"]
+regex = ["integer", "dep:combine", "dep:anyhow"]
 internal-keycache = ["lazy_static", "fs2", "bincode"]
 
 # Experimental section
-experimental = []
+experimental = ["experimental-multi_bit_pbs"]
+experimental-multi_bit_pbs = []
```

Comment:
These features do not exist anymore, i guess this comes from a rebasing

Reply:
correct, went wrong during a rebase merge. removing